contrib/build-push-ceph-container-imgs.sh

```diff
@@ -245,13 +250,13 @@ function push_ceph_imgs_latest {
     local_tag=${CONTAINER_REPO_ORGANIZATION}/daemon-base:${RELEASE}-${BRANCH}-centos-${CENTOS_RELEASE}-${HOST_ARCH}
     full_repo_tag=${CONTAINER_REPO_HOSTNAME}/${CONTAINER_REPO_ORGANIZATION}/ceph:${RELEASE}-centos-${CENTOS_RELEASE}-${HOST_ARCH}-devel
     branch_repo_tag=${CONTAINER_REPO_HOSTNAME}/${CONTAINER_REPO_ORGANIZATION}/ceph:${BRANCH}
-    sha1_repo_tag=${CONTAINER_REPO_HOSTNAME}/${CONTAINER_REPO_ORGANIZATION}/ceph:${SHA1}
+    sha1_os_flavor_repo_tag=${CONTAINER_REPO_HOSTNAME}/${CONTAINER_REPO_ORGANIZATION}/ceph:${SHA1}:${OS}-${CENTOS_RELEASE}:${OSD_FLAVOR}
```

Comment:
using extra `:`  in the tag name isn't supported

Reply:
aah, I see. We wanted to identify a container based on shaman flavor(OSD_FLAVOR) since sha1 alone does not act as a unique identifier.
reference from irc:
```
otoh, would be great to have the OS there as well. evenutally I'd like to get leap containers as well. 
7:14 PM <kefu> Kefu Chai os_type-os_ver-flavor.
7:15 PM <SebastianW> Sebastian Wagner +1
7:37 PM <ideepika> Deepika Upadhyay kefu Sebastian: would it be sha1-os_type-os_ver-flavor as a whole, since we would want to a uniqueID for a particular branch? 
7:38 PM SebastianW
7:38 PM <SebastianW> Sebastian Wagner yes
7:38 PM <SebastianW> Sebastian Wagner we need to identify the branch. sha1 was the least problematic
7:38 PM <SebastianW> Sebastian Wagner otherwise you'll end up testing the wrong build. 
7:39 PM so yes
7:39 PM sha1-os-ver-flavor
```

Reply:
so you can just add the OSD_FLAVOR to the tag but with a dash or underscore between both value (ie: not `:`)

Reply:
@dsavineau would it be okay to use include `sha1-centos-8-flavor` tag as Sebastian mentioned in future they might want to use leap container as well.
Updated to dash from colon

Reply:
> as Sebastian mentioned in future they might want to use leap container as well.

There's no reason to have multiple container images with the same Ceph version (ie: with different GNU/Linux base image). Container image delivers software (ceph in this case) not OS.